### PR TITLE
Add Evidence Banner to teacher home page, copy change to Premium page

### PR DESCRIPTION
**Pull request by cissyxyu**
## WHAT
Add a new Evidence Banner to the teacher home page, replacing the Diagnostics Banner. This banner will only show for teachers who are teaching grades 8th through 12th. 
I also made a small copy change to the Premium page.

## WHY
We want to promote Evidence activities to high school teachers.

## HOW
Add a new banner with new styling. Also add a back end method to determine whether a teacher teaches grades 8th through 12th.

### Screenshots
Screen Shot 2022-10-19 at 2 41 58 PM

Screen Shot 2022-10-26 at 12 40 41 PM

### Notion Card Links
https://www.notion.so/quill/Launch-Banner-Reading-for-Evidence-f69da23155624b2aa3b40ae4165082df
https://www.notion.so/quill/Update-Pricing-Page-to-Specify-that-Teacher-Payments-are-via-Credit-Card-Small-Design-Change-3e0c36e07ae94c62975afd9383e5d15a

PR Checklist | Your Answer
------------ | -------------
Have you added and/or updated tests? |  YES
Have you deployed to Staging? | Not yet - deploying now!
Self-Review: Have you done an initial self-review of the code below on Github? | Yes
Spec Review: Have you reviewed the spec and ensured this meets requirements and/or matches design mockups? | Yes

### CODE REVIEW

File: services/QuillLMS/app/models/concerns/teacher.rb [hunk 1]
```diff
@@ -322,6 +322,10 @@ def classroom_minis_info
     info
   end
 
+  def teaches_eighth_through_twelfth?
+    return classrooms_i_teach.map(&:grade).compact.any? { |grade| grade.to_i <= 12 && grade.to_i >= 8 }
```
Comment (brendanshean) [inline services/QuillLMS/app/models/concerns/teacher.rb]: [between?](https://ruby-doc.org/core-2.4.0/Comparable.html#method-i-between-3F) might be more readable here:

`classrooms_i_teach.map(&:grade).compact.any? { |grade| grade.to_i.between?(8,12) }`

Comment (emilia-friedberg) [reply services/QuillLMS/app/models/concerns/teacher.rb]: "Twelfth" is a word that will always look spelled wrong to me even though it is in fact correct.

**Review by emilia-friedberg — APPROVED** [review]

**Review by brendanshean — APPROVED** [review]
Looks good to me.

File: services/QuillLMS/app/models/concerns/teacher.rb (hunk 1 above)
Comment (cissyxyu) [reply services/QuillLMS/app/models/concerns/teacher.rb]: @emilia-friedberg I know!!! I had to google "twelfth" to make sure.

**Pull request merged**